src/controlpotmeter.h

```diff
@@ -74,6 +74,7 @@ class ControlPotmeter : public ControlObject {
     Q_OBJECT
   public:
     ControlPotmeter(ConfigKey key, double dMinValue = 0.0, double dMaxValue = 1.0,
+                    double dNeutralValue = 0.0,
```

Comment:
New values, in a row of parameters featuring a default have to be at the end. 
Otherwise, you have no control, that the calling code does not mix the parameter. I have learned it the hard way, introducing the default value. 

Maybe it is time to introduce a parameter class, that sets the default values in its constructor and features a setter for each parameter.